src/server/game/Entities/Player/Player.h

```diff
@@ -2949,6 +2952,8 @@ class Player : public Unit, public GridObject<Player>
     bool _wasOutdoor;
 
     PlayerSettingMap m_charSettingsMap;
+
+    time_t m_creationDate;
```

Comment:
Is there no way to use chrono for this? :(

Reply:
The server requires us to send the milliseconds since character creation for the "average" achievement criteria, so I thought I'd be storing the Unix timestamp of character creation as reference. Also I'm not familiar with std::chrono, maybe you've got a suggestion and I can give it a try?

Reply:
I will try to change it to using std chrono time_point to store the date if that's what you would prefer?

Reply:
I don't really have time to look at it right now. But std chrono is always preferred when possible

Reply:
Alright I will give it a try and provide an update